azext_iot/monitor/models/arguments.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import AzCliCommand
+from azext_iot.common.utility import init_monitoring
+from azext_iot.monitor.models.enum import Severity
+
+
+class TelemetryArguments:
+    def __init__(
+        self,
+        cmd: AzCliCommand,
+        timeout: int,
+        properties: list,
+        enqueued_time: int,
+        repair: bool,
+        yes: bool,
+    ):
+        (enqueued_time, unique_properties, timeout_ms, output) = init_monitoring(
+            cmd, timeout, properties, enqueued_time, repair, yes
```

Comment:
nit: can we use keyword args when calling init_monitoring

Reply:
I'm not sure if I can see the benefit of moving this to kwargs, how strongly do you feel about this?

I also feel like the `init_monitoring` call might need some refactoring, its slightly odd that it does stuff like set defaults and return values with the same name as the input params. I was even thinking about putting it somewhere in the `monitor/` path

Reply:
The benefit when used is order of args won't matter which is increased in value when the number of args for a function increase. One less thing to worry about.

This one is just a nit - I am perfectly fine if no change is made.

Yes, I agree on refactoring but not in this PR please.

Reply:
Just so we make sure we are talking about the same thing

I meant 

init_monitoring(cmd=cmd, timeout=timeout, properties=properties, enqueued_time=enqueued_time, repair=repair, yes=yes)

**Not** usage of **kwargs expansion

Reply:
OHHH ok, I'm in favor of that, I'll make that fix